share/spice/thumbtack/thumbtack.js

```diff
@@ -0,0 +1,41 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_thumbtack = function(api_result){
+
+        // Don't show anything if we weren't able to return 2 or more services.
+        // In future iterations, we should support a different view in the case
+        // of a single result.
+        if (!api_result || api_result.error) {
```

Comment:
Please check for the length of `data` as well. I don't know how often this happens, but I could imagine a scenario where there are no plumbers in Philly and `data` might have 0 elements in it.

```javascript
if (!api_result || !api_result.data || !api_result.data.length)
```